wiki/Play_style/Alternating/de.md

```diff
@@ -0,0 +1,28 @@
+---
+tags:
+  - keyboard
+  - tap
+  - alternate
+  - alt
+  - play style
+  - Tastatur
+  - Spielstil
+---
+
+# Alternating
+
+*Nicht zu verwechseln mit [Double-Tapping](/wiki/Play_style/Double_tapping).*
+
+**Alternating** (auf Deutsch *abwechselnd*) ist ein Tippstil für [osu!](/wiki/Game_mode/osu!), bei dem Spieler durch das Alternieren zwischen zwei eingestellten Tasten auf ihrer Tastatur (standardmäßig `Z` und `X`) Circles klicken, Slider verschieben und Spinner spinnen, um dadurch jedes zweite [Hit-Objekt](/wiki/Hit_object) zu treffen. Der Name kommt durch den Wechsel zwischen den Tasten. Dieser Stil ist am meisten verbreitet unter erfahrenen Spielern und verlangt eine der schwierigsten Anpassungen.
+
+## Verwendung
+
+Alternating hat eine große Vielfalt an Anwendungen und seine Formen können von Spieler zu Spieler variieren. Zum Beispiel wechseln viele Spieler, die abwechselnd die Tasten drücken, zwischen `Z` und `X` (auf dem deutschen Tastaturlayout wählen die meisten `Y` und `X`) auf jeder zweiten Note (oder Hit-Object), während andere hingegen die Tasten nach jedem zweiten oder dritten Hit-Objekt wechseln. Es sollte erwähnt werden, dass manche Spieler die meiste Zeit alternieren, aber manchmal dieselbe Taste zweimal drücken, wenn ein Hit-Objekt den Spieler überrascht. Das liegt alles an den persönlichen Vorlieben.
```

Comment:
As you can see, I've added an explanation about the usage of Y and X instead of Z and X because these two are miles away from each other (due to QWERTZ).